docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLpNorm.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Samples.Dynamic
+{
+    class NormalizeLpNorm
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+            var samples = new List<DataPoint>()
+            {
+                new DataPoint(){ Features = new float[4] { 1, 1, 0, 0} },
+                new DataPoint(){ Features = new float[4] { 2, 2, 0, 0} },
+                new DataPoint(){ Features = new float[4] { 1, 0, 1, 0} },
+                new DataPoint(){ Features = new float[4] { 0, 1, 0, 1} }
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+            // NormalizeLpNorm normalize rows individually by rescaling them to unit norm.
+            // Performs the following operaion on a row X:  Y = (X - M(X)) / D(X) 
+            // where M(X) is scalar value of mean for current row,
+            // and D(X) is scalar value of selected `norm` parameter .
+            var approximation = mlContext.Transforms.NormalizeLpNorm("Features", norm: LpNormNormalizingEstimatorBase.NormFunction.L1, ensureZeroMean: true);
```

Comment:
>ensureZeroMean [](start = 135, length = 14)

What does EnsureZeroMean do? Subtract the mean?

Reply:
yes, added it to comment above.

---
In reply to: [273740225](https://github.com/dotnet/machinelearning/pull/3232#discussion_r273740225) [](ancestors = 273740225)

Reply:
Let's move parameter details to xml docstring.

---
In reply to: [273741392](https://github.com/dotnet/machinelearning/pull/3232#discussion_r273741392) [](ancestors = 273741392,273740225)